src/plays/star-rating/StarRating.js

```diff
@@ -0,0 +1,82 @@
+import PlayHeader from "common/playlists/PlayHeader";
+import { useState } from "react";
+import { FaStar } from "react-icons/fa";
+import "./styles.css";
+
+// WARNING: Do not change the entry componenet name
+function StarRating(props) {
```

Comment:
As it's intended to be a component, I would suggest creating another component and exposing a selection method like `onSelectionChanged` and using it here. So the consumer can directly be benefitted from it

Reply:
Do you mean, create a global state

Reply:
Lets ignore this comment. It would be helpful if you create a reusable component or npm package.